octodns/provider/cloudflare.py

```diff
@@ -44,6 +48,16 @@ class CloudflareProvider(BaseProvider):
         #
         # See: https://support.cloudflare.com/hc/en-us/articles/115000830351
         cdn: false
+
+    Note: The "proxied" flag of "A", "AAAA" and "CNAME" records can be managed
+          via the YAML provider like so:
+              subdomain:
```

Comment:
I think `name` would be a little more consistent with the octoDNS terminology elsewhere.